_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -141,55 +141,58 @@ $shadow: transparentize(palette(grayscale, black-t), .5);
 // ----------------------------
 // #TYPOGRAPHY
 // ----------------------------
+
+$ux-pattern-library-font-path: '../fonts' !default;
```

Comment:
@Nickersoft you asked for my thoughts and almost immediately merged.

I suggested using a prefixed name here so as not to conflict with any other user variables, or the variables of other libraries. For example, FontAwesome uses the `fa-` prefix for most of its variables. Had Dave chosen to use `$font-path` instead of `$fa-font-path`, we would have a problem with integrating these two libraries. I *highly* recommend adding a prefix to avoid this problem down the road.